driver-scala/src/main/scala/org/mongodb/scala/model/Aggregates.scala

```diff
@@ -507,7 +542,8 @@ object Aggregates {
    *
    * @param collector A search collector.
    * @param options Optional `\$search` pipeline stage fields.
-   * Specifying `null` is equivalent to specifying `SearchOptions.defaultSearchOptions`.
+   * Specifying `null` is equivalent to specifying `SearchOptions.defaultSearchOptions`
```

Comment:
As previous in Scala I would leave this out.

Reply:
Done.